layouts/shortcodes/added-in.html

```diff
@@ -2,7 +2,7 @@
 {{ $this := .Params.this }}
 
 {{ if $this }}
-  <span>**[New in this version]**</span>
```

Comment:
These changes are needed, otherwise it displays the markdown in the federation docs, rather than the HTML

Reply:
hmm.. that's surprising. What version of hugo?

Reply:
It happened with whatever version of hugo CI was using.  It worked fine in the CS bits.  I assume it's because it got substituted in, and then the file got included in the main file, and then reprocessed.  But it didn't work for the federation doc.

Reply:
right, yea. `{{< shortcode >}}` is a literal while `{{% shortcode %}}` is interpreted - we need to use angle brackets to get the inline stuff to work, annoyingly.

Reply:
This is fun though (and not worth fixing in this PR, I think):

s2s spec:
![image](https://user-images.githubusercontent.com/1190097/170805391-fb4ff94d-c675-4707-90e7-2aa2ea094b90.png)

c2s spec:
![image](https://user-images.githubusercontent.com/1190097/170805394-751d8788-50a5-4b53-87b1-aacf7947887e.png)

The code blocks are different. I think it's because of the table structure, but not really sure.